js/apps/system/_admin/aardvark/APP/react/public/assets/src/mode-aql.js

```diff
@@ -88,7 +88,7 @@ var TextHighlightRules = require("./text_highlight_rules").TextHighlightRules;
 var AqlHighlightRules = function() {
 
     var keywords = (
-        "for|return|filter|search|sort|limit|let|collect|asc|desc|in|into|insert|update|remove|replace|upsert|options|with|and|or|not|distinct|graph|shortest_path|outbound|inbound|any|all|none|aggregate|like|k_shortest_paths|k_paths|prune|window"
+        "for|return|filter|search|sort|limit|let|collect|asc|desc|in|into|insert|update|remove|replace|upsert|options|with|and|or|not|distinct|graph|shortest_path|outbound|inbound|any|all|none|aggregate|like|k_shortest_paths|k_paths|all_shortest_paths|prune|window"
```

Comment:
just want to double check in this case that this is indeed all that is required for frontend AQL configuration

Reply:
We also do have:
```
    aqlKeywords: 'for|return|filter|search|sort|limit|let|collect|asc|desc|in|into|' +
      'insert|update|remove|replace|upsert|options|with|and|or|not|' +
      'distinct|graph|shortest_path|outbound|inbound|any|all|none|aggregate|like|count',
```

In: 
`frontend/js/views/spotlightView.js`

Reply:
But of course, best solution would be to have a unified way of using this ... and generated if possible.

Reply:
thank you! addressed in https://github.com/arangodb/arangodb/pull/16407/commits/e9b99ff481a02e06356219436d4ea6af2908a610